package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "gpii-electron",
+  "version": "1.0.0",
+  "description": "Electron application and libraries to run the GPII as a native user space application on the desktop",
+  "main": "main.js",
+  "scripts": {
+    "start": "electron main.js",
+    "test": "electron tests.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/gpii/gpii-electron.git"
+  },
+  "keywords": [
+    "Electron",
+    "quick",
+    "start",
+    "tutorial"
+  ],
+  "devDependencies": {
+    "electron": "1.4.1",
+    "eslint-config-fluid": "^1.0.0",
+    "fluid-grunt-eslint": "18.1.2",
+    "grunt": "1.0.1",
+    "grunt-jsonlint": "1.1.0",
+    "grunt-shell": "1.3.0",
+    "request": "^2.69.0",
```

Comment:
This and electron are in the wrong block since they are part of the implementation. Also, our policy is to use exact versions throughout.